reana-ui/src/components/WorkflowActionsPopup.js

```diff
@@ -80,7 +82,20 @@ export default function WorkflowActionsPopup({ workflow, className }) {
     });
   }
 
-  if (!isDeleted) {
+  if (isRunning) {
+    menuItems.push({
+      key: "stop",
+      content: "Stop workflow",
+      icon: "stop",
```

Comment:
I like the `stop` icon in the actions popup, because `pause` usually refers to something that can be resumed later, and that's not the case here.
For this reason I would also prefer changing the `pause` icon in the workflow list to something else. Using `stop circle outline` would work, with the small downside that in general the `stop` icon is used only as an action icon, rather than as a status icon. However, IMHO it's better than conveying the information that a stopped workflow can be resumed.

Reply:
Updated to use the `stop` icon also in the workflow list